### PR TITLE
FEAT:Monitoring System Page

### PR DESCRIPTION
**Pull request by DavMovsesyan2002**

### CODE REVIEW

File: src/components/OrDivider/Divider.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const Divider = () => (
+  <div
+    className="relative my-8 h-px w-full bg-gray-light
+              after:absolute after:left-0 after:right-0 after:-top-3.5 after:mx-auto
+              after:w-14 after:bg-white after:text-center after:text-base after:text-black-light"
+  />
+)
```
Comment (ArthurO2002) [inline src/components/OrDivider/Divider.tsx:9]: Can we use OrDivider component with condition that will remove `or`

Comment (ArthurO2002) [reply src/components/OrDivider/Divider.tsx:9]: Rename that component to `Divider`

File: src/pages/monitoring-systems/index.tsx [hunk 2]
```diff
@@ -41,14 +55,42 @@ const MonitoringSystems = () => (
...
             {tab.content}
           </Tab.Panel>
         ))}
+        <Button
+          variant="outlined"
+          size="sm"
+          className="mb-20"
+        >
+          Download Graph and Data
+        </Button>
```
Comment (ArthurO2002) [inline src/pages/monitoring-systems/index.tsx:93]: Button should not be there

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

**Review by ArthurO2002 — APPROVED** [review]

File: src/components/Divider/Divider.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+interface DividerProps {
+  content?: boolean
+}
+
+export const Divider = ({ content }: DividerProps) => (
+  <div
+    className={`relative my-8 h-px w-full bg-gray-light
+              after:absolute after:left-0 after:right-0 after:-top-3.5 after:mx-auto
+              after:w-14 after:bg-white after:text-center after:text-base after:text-black-light ${
+                content && 'after:content-["or"]'
```
Comment (rikarsen) [inline src/components/Divider/Divider.tsx:12]: please use divide from tailwind, and orDivider for or

**Review by gor017 — DISMISSED** [review]

File: src/pages/profile/monitoring-systems/index.tsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+              />
+            </div>
+            <div className=" mr-5 w-[124px]">
+              <Autocomplete
+                items={years}
+                placeholder="Region"
+              />
+            </div>
+          </div>
+          <div className="mb-5 xl:w-[124px]">
```
Comment (rikarsen) [inline src/pages/profile/monitoring-systems/index.tsx:98]: use sm for small size and fl for full width

**Review by rikarsen — CHANGES_REQUESTED** [review]

**Review by ArthurO2002 — APPROVED** [review]

**Review by ArthurO2002 — DISMISSED** [review]

**Review by ArthurO2002 — DISMISSED** [review]

**Review by rikarsen — APPROVED** [review]

**Pull request merged**